admin-ui/app/locales/en/translation.json

```diff
@@ -390,6 +390,8 @@
     "saml1_uri": "Saml1 URI",
     "saml2_uri": "Saml2 URI",
     "saml_acr": "SAML ACR",
+    "bindDN": " Bind DN",
+    "primaryKey": " Primary Key",
     "scope_type": "Scope type",
```

Comment:
Defining duplicate translation keys with inconsistent casing and un-normalized string values is bad because it violates the DRY principle by creating multiple keys for the same semantic concept, which leads to maintenance issues and potential inconsistencies if one key is updated and the other is not.